scripts/diagnostics.txt

```diff
@@ -972,6 +972,9 @@ subsystem Meta
 error TooManyErrors "too many errors emitted, stopping now [--error-limit=]"
 warning unknown-warning-option UnknownWarningOption "unknown warning option '{}'"
 
+subsystem Tidy
```

Comment:
Do you mean to have a `diagnostics.txt` in the slang-tidy, or have a whole diagnostic system for the slang-tidy tool?

Reply:
The fact that those new diagnostics are not checked using slang, but with slang-tidy, makes the docs CI to fail. Since it tries to trigger the warning with the diagnostics docs example using slang.

Reply:
Yeah, that's part of why I think it should be a separate file (like diagnostics.txt), but I'm not entirely sure what it should look like. Perhaps analysis "warnings" in slang-tidy don't need to be in a separate file at all but could just put their message inline with the source code.

Reply:
I have moved the diagnostics to be inline in the visitor code, and removed them from the `diagnostics.txt` file.